samples/nrf9160/multicell_location/prj.conf

```diff
@@ -34,9 +34,9 @@ CONFIG_MULTICELL_LOCATION_MAX_NEIGHBORS=10
 # CONFIG_MODEM_JWT=y
 
 # nRF Cloud location service configuration
-# CONFIG_MULTICELL_LOCATION_SERVICE_NRF_CLOUD=y
-# CONFIG_NRF_CLOUD_REST=y
-# CONFIG_MODEM_JWT=y
+CONFIG_MULTICELL_LOCATION_SERVICE_NRF_CLOUD=y
```

Comment:
Hold on with this change. I want to understand why this is needed as it hasn't been so far. I've got a comment elsewhere on this:
https://github.com/nrfconnect/sdk-nrf/pull/6168#discussion_r755730741

Reply:
removed from this PR